cmd/cmd.go

```diff
@@ -66,7 +66,7 @@ func init() {
 	// will be global for your application.
 
 	rootCmd.PersistentFlags().StringVar(&kubeConfig, "kubeconfig", "", "kubernetes config path")
-	rootCmd.PersistentFlags().StringVar(&sourceStorageClass, "sourcestoraageclass", "", "source storageclass from which all PVC need to be migrated")
+	rootCmd.PersistentFlags().StringVar(&sourceStorageClass, "sourcestorageclass", "", "source storageclass from which all PVC need to be migrated")
```

Comment:
```suggestion
	rootCmd.PersistentFlags().StringVar(&sourceStorageClass, "source-storageclass", "", "source storageclass from which all PVC need to be migrated")
```

Reply:
already part of #27 as diff commit but can make separate PR